package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "tabletop.js",
-  "version": "1.0.0",
+  "name": "tabletop-boardgames",
+  "version": "1.0.2",
```

Comment:
It has to be x.x.x can you be more specific? I've already published this so I don't think we can make the version number less